code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -469,6 +469,15 @@
 				user.visible_message("[user] has secured [src]'s floor casters.", \
 						 								"<span class='notice'>You have secured [src]'s floor casters.</span>")
 				anchored = TRUE
+	if(opened)

```

Comment:
why not just do 
```
	else
		..()
```

instead of having all of this?

Reply:
To avoid people rigging the garbage bins, transporting them with a RCS, etc

Reply:
^
Unless you'd also like the full boobytrapping functionality to be added, @Fox-McCloud , this is really just to allow people to dump stuff into the can.

Reply:
Even if that's the case, it's best to have a var that controls that behavior on the parent, then you set that var to `TRUE` or `FALSE` on children that you don't want to/want to have that behavior.

That said, I do think the rigging and stuff should be retained. Watching someone shock themselves from just opening a trash can would be hilarious.

Reply:
Grayshirt: "I'm going to throw away my trash like a human being"
*opens trash can*
***BOOM***
Grayghost: "WHYYYYYYY"